src/XRP/model/xrp-escrow-finish.ts

```diff
@@ -20,22 +21,34 @@ export default class XrpEscrowFinish {
   public static from(
     escrowFinish: EscrowFinish,
     xrplNetwork: XrplNetwork,
-  ): XrpEscrowFinish | undefined {
+  ): XrpEscrowFinish {
     const owner = escrowFinish.getOwner()?.getValue()?.getAddress()
     if (!owner) {
-      return undefined
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'EscrowFinish protobuf is missing required `owner` field.',
+      )
     }
     const ownerXAddress = XrpUtils.encodeXAddress(
       owner,
       undefined,
       xrplNetwork == XrplNetwork.Test || xrplNetwork == XrplNetwork.Dev,
```

Comment:
if you wanted to make these `===` i wouldn't be pressed

Reply:
lol maybe Mayukha can explain to me the nuances of "pressed" in the context of gen Z?

Reply:
Unfortunately I'm only about half gen Z so I don't qualify for the gen Z dictionary